tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3332,18 +3332,6 @@ declare namespace ts {
              * Last version that was reported.
```

Comment:
Unrelated to your PR, but this is somehow the first time that I've noticed that our dts emitter emits jsdoc on things marked private. Wow! (Surely that behavior isn't desirable?)